packages/recomponents/src/components/r-repeater/r-repeater.md

```diff
@@ -0,0 +1,48 @@
+# Repeater
+
+This is an abstract component to help you render data in a naked table. It is notably used by `r-grid` under the hood, which is probably the component you want to use :)
```

Comment:
This isn't strictly correct, I believe `r-repeater` is intended to be used on its own outside of Grid